pootle/static/js/editor/utils/strings.js

```diff
@@ -11,12 +11,12 @@ export const Formats = {
   PLURR: 'PLURR',
 };
 
-/**
- * Returns the string format detected for `value`.
- */
 export function detectFormat(value) {
   if (value.search(/{[^{}]*:.*?}/) !== -1) {
     return Formats.PLURR;
+  } else if (value.search(/{[^ ]*}/) !== -1) {
```

Comment:
I think this regular expression will also match `{}`. While technically it is allowed in Plurr, let's make sure we test this locally on such a string and make sure it doesn't break the UI.

Reply:
I'll check

Reply:
<img width="982" alt="image" src="https://user-images.githubusercontent.com/26126311/81953345-8cbd2480-95d5-11ea-977a-545ce9be7708.png">

The UI does not break, but the specific case is not addressed. As discussed a separated issue has been created to work on this.